test/unit/cluster_resource_discovery_test.rb

```diff
@@ -50,7 +50,7 @@ def test_prunable_namespaced_resources
     kinds = crd.prunable_resources(namespaced: true)
 
     assert_equal(kinds.length, 25)
-    %w(ConfigMap CronJob Deployment).each do |expected_kind|
+    %w(ConfigMap CronJob Deployment batch/v1/Job).each do |expected_kind|
```

Comment:
Shouldn't this just be "Job"? Otherwise of course L54 will only ever find one matching entry, before or after this PR. I guess the matcher needs to get more specific than just `include?` since it will also match "CronJob".

Reply:
@KnVerey the point of the test is to assert that the version override is working and a specific Job API is returned ("v1" not v1beta1 or v2alpha1)

Reply:
@sco11morgan  The name of this test is  vague, it was originally designed to ensure that a few obvious Kinds were present. How would you feel about adding an additional block to test for full group/version/kind matches:

```
%w(batch/v1/Job batch/v1beta1/cronjob).each do |expected_gvk|
  assert kinds.include?(expected_gvk)
end

```

Reply:
👍 moved the assertions to its own test